src/actionCreators/resourceHelpers.js

```diff
@@ -208,6 +203,15 @@ const newProperty = (subject, propertyTemplate, noDefaults, errorKey) => (dispat
   return property
 }
 
+export function defaultValuesFor(property) {
+  return property.propertyTemplate.defaults.map((defaultValue) => {
+    if (property.propertyTemplate.type === 'uri') {
+      return newUriValue(property, defaultValue.uri, defaultValue.label)
+    }
+    return newLiteralValue(property, defaultValue.literal, defaultValue.lang)
+  })
+}
```

Comment:
a new extraction refactor function that fetches the default values for a property template, extracted here so it can be used when both adding a new property and expanding a property